client/container_create.go

```diff
@@ -39,6 +39,9 @@ func (cli *Client) ContainerCreate(ctx context.Context, config *container.Config
 	if err := cli.NewVersionError(ctx, "1.44", "specify health-check start interval"); config != nil && config.Healthcheck != nil && config.Healthcheck.StartInterval != 0 && err != nil {
 		return response, err
 	}
+	if err := cli.NewVersionError("1.44", "specify mac-address per network"); hasEndpointSpecificMacAddress(networkingConfig) && err != nil {
```

Comment:
`NewVersionError` returns an error like `"%q requires API version %s, but the Docker daemon API version is %s",` so in this case this would read as `specify mac-address per network requires API version...` which reads odd, I think we want to say `specifying mac-address per blah blah blah` 

Obviousely, this is a huge nit and not a blocker